suites/pacific/upgrades/tier-2_upgrade_test-multi-path-upgrade-to-5.1-latest.yaml

```diff
@@ -59,16 +59,23 @@ tests:
               command: upgrade_all
         5.1:
           config:
-            command: start
-            service: upgrade
-            base_cmd_args:
-              verbose: true
-            benchmark:
-              type: rados                      # future-use
-              pool_per_client: true
-              pg_num: 128
-              duration: 10
-            verify_cluster_health: true
+            verify_cephadm_containers: True
+            ansi_config:
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              copy_admin_key: true
+              dashboard_enabled: True
+              dashboard_admin_user: admin
+              dashboard_admin_password: p@ssw0rd
+              grafana_admin_user: admin
+              grafana_admin_password: p@ssw0rd
+              containerized_deployment: true
+              upgrade_ceph_packages: True
+              cephfs_pools:
+                - name: "cephfs_data"
+                  pgs: "8"
+                - name: "cephfs_metadata"
+                  pgs: "8"
```

Comment:
Are you sure about this configuration? I believe it would put the cluster health in warning state.

Reply:
Changes done, increased pg number